articles/cms/wordpress/troubleshoot.md

```diff
@@ -4,10 +4,22 @@ description: This page explains common troubleshooting issues with WordPress.
 
 # WordPress Troubleshooting
 
+## I'm seeing a message "This account does not have an email associated..." that stops me from logging in.
+
+If you get this error, make sure you are requesting an email from each provider in the Auth0 Dashboard under Connections -> Social (expand each provider). Take into account that not all providers return email addresses for users (e.g. Twitter). If this happens, you can always add an Email address to any logged in user through the Auth0 Dashboard (or API). See Users -> Edit.
+
+## I'm getting a "Failed cross origin authentication" or "No verifier returned from client" error in my error logs or when logging in.
+
+Check your "Allowed Callback URLs" and "Allowed Origins (CORS)" fields in the Client settings for your WordPress site to make sure those are correct. If you're using a Chromium-based browser, review our [docs page on cross-origin authentication](https://auth0.com/docs/cross-origin-authentication#limitations-of-cross-origin-authentication) to make sure you don't have third-party cookies turned off.  
```

Comment:
Relative URL `[docs page on cross-origin authentication](/cross-origin-authentication#limitations-of-cross-origin-authentication")`